lib/services/header/index.js

```diff
@@ -159,6 +159,13 @@ HeaderService.prototype._setGenesisBlock = function(callback) {
     merkleRoot: '730f0c8ddc5a592d5512566890e2a73e45feaa6748b24b849d1c29a7ab2b2300'
   };
 
+  if (this.GENESIS_HASH === 'ec42fa26ca6dcb1103b59a1d24b161935ea4566f8d5736db8917d5b9a8dee0d7') {
```

Comment:
Should there be another section similar to this for configuring the Testnet parameters as well?

https://github.com/floblockchain/flo/blob/2fe0451/src/chainparams.cpp#L281


Where is testnet currently configured and how does Bitcoin handle the different values between networks?

Reply:
It looks like making these changes to the header service is actually not required at all. I had initially thought this change was required, but upon testing without the change it works just fine, so I will remove this extra logic.

It doesn't look like the Bitcoin implementation of this does anything to handle different values between networks either, so I will just match that. https://github.com/bitpay/bitcore-node/blob/master/lib/services/header/index.js#L145

Good question and catch @bitspill

Reply:
Interesting, those fields must go unused in this library I guess unless that's an upstream issue